tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/XUnitTests.cs

```diff
@@ -36,211 +32,207 @@ public XUnitTests(ITestOutputHelper output)
             SetServiceVersion("1.0.0");
         }
 
-        [SkippableTheory]
-        [MemberData(nameof(PackageVersions.XUnit), MemberType = typeof(PackageVersions))]
-        [Trait("Category", "EndToEnd")]
-        [Trait("Category", "TestIntegrations")]
-        public async Task SubmitTraces(string packageVersion)
+        public virtual async Task SubmitTraces(string packageVersion)
         {
-            List<MockSpan> spans = null;
             string[] messages = null;
+            SetEnvironmentVariable(ConfigurationKeys.CIVisibility.Enabled, "1");
+            SetEnvironmentVariable(ConfigurationKeys.DebugEnabled, "1");
```

Comment:
```suggestion
```

Reply:
done in [5dc9c9d](https://github.com/DataDog/dd-trace-dotnet/pull/5634/commits/5dc9c9d4f26816d8a22caa72ceeeb3807286fd08)